grakn-test/src/test/java/ai/grakn/test/engine/controller/UserControllerTest.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Grakn - A Distributed Semantic Database
+ * Copyright (C) 2017  Grakn Labs Ltd
+ *
+ * Grakn is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Grakn is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Grakn. If not, see <http://www.gnu.org/licenses/gpl.txt>.
+ */
+
+package ai.grakn.test.engine.controller;
+
+import ai.grakn.Grakn;
+import ai.grakn.GraknGraph;
+import ai.grakn.GraknSession;
+import ai.grakn.GraknTxType;
+import ai.grakn.engine.controller.UserController;
+import ai.grakn.engine.factory.EngineGraknGraphFactory;
+import ai.grakn.engine.user.UsersHandler;
+import ai.grakn.factory.SystemKeyspace;
+import ai.grakn.test.SparkContext;
+import com.jayway.restassured.response.Response;
+import mjson.Json;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Ignore;
+import org.junit.Rule;
+import org.junit.Test;
+import org.mockito.stubbing.Answer;
+
+import static com.jayway.restassured.RestAssured.get;
+import static com.jayway.restassured.RestAssured.given;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+@Ignore
+public class UserControllerTest {
```

Comment:
It looks like the merge commit has accidentally created a duplicate `UserControllerTest`.